packages/ember-views/lib/system/build-component-template.js

```diff
@@ -26,7 +26,8 @@ export default function buildComponentTemplate({ component, layout, isAngleBrack
     // element. We use `manualElement` to create a template that represents
     // the wrapping element and yields to the previous block.
     if (tagName !== '') {
-      var attributes = normalizeComponentAttributes(component, isAngleBracket, attrs);
+      var canChangeType = canChangeTypeAfterRender(attrs);
```

Comment:
I agree, but that will make it more difficult to unit test in good browsers. See https://github.com/emberjs/ember.js/pull/12351/files#diff-419c3732f6d0ff04c367a6fe47525974R48